--- a/fuse-archive.cc	2025-08-20 00:41:52.000000000 +0000
+++ b/fuse-archive.cc	2025-11-06 16:25:51.842296926 +0000
@@ -2488,10 +2488,6 @@
   if (struct stat z; fstat(g_archive_fd, &z) != 0) {
     PLOG(ERROR) << "Cannot stat " << Path(g_archive_path);
     throw ExitCode::CANNOT_OPEN_ARCHIVE;
-  } else if (FileType const ft = GetFileType(z.st_mode); ft != FileType::File) {
-    LOG(ERROR) << "Archive " << Path(g_archive_path)
-               << " is not a regular file: It is a " << ft;
-    throw ExitCode::CANNOT_OPEN_ARCHIVE;
   } else {
     g_archive_size = z.st_size;
     LOG(DEBUG) << "Archive file size is " << g_archive_size << " bytes";
